styles/index.css

```diff
@@ -0,0 +1,83 @@
+button {
+    background: lightblue;
+    padding: 10px;
+    border-radius: 25px;
+}
+
+#weatherFrame {
```

Comment:
It looks like you leaned pretty hard on ids here, which does make it easier to style things but slightly hampers re-usability of styles.